### DISCUSSION TITLE
NodeJS Update on new RC

### DISCUSSION THREAD

**Discussion by Lucas-Gruber** [directus/directus#6728] [Archive]
Hello! I have the Version RC.81, and i would like to update to the newest version (at this time 83). But i don't know what i have to do to update Directus without clearing data or anything else. I hope anyone can help me! :)
\
\
\
\
ps: sorry for the bad english, i'm a german potato 😅

**Comment by RageBrahimi** [comment]
If you run `npm install directus@latest` it will install the latest version for you. Once you've done that, be sure to upgrade your DB via the CLI using the instructions on [Upgrade the Database](https://docs.directus.io/reference/command-line-interface/#upgrade-the-database). 

I'd always recommend a backup before you proceed with the above.

**Reply by Lucas-Gruber** [reply]
Alright thanks! And is it enough if I copy the Directus folder or do I have to backup the DB too?

**Reply by martinemmert** [reply]
You should backup the data of your database as well. (a SQL Dump should be enough)

**Reply by Lucas-Gruber** [reply]
Okay thank you very much! :)
